docs/content/en/blog/articles/minio-backup/index.md

```diff
@@ -0,0 +1,268 @@
+
+```
+CREATE KEYSPACE medusa_test  WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};
+USE medusa_test;
+CREATE TABLE users (email text primary key, name text, state text);
+insert into users (email, name, state) values ('alice@example.com', 'Alice Smith', 'TX');
+insert into users (email, name, state) values ('bob@example.com', 'Bob Jones', 'VA');
+insert into users (email, name, state) values ('carol@example.com', 'Carol Jackson', 'CA');
+insert into users (email, name, state) values ('david@example.com', 'David Yang', 'NV');
+```
+
+And copy it into the Cassandra pod (the StatefulSet one, which contains <code>-<strong>sts</strong>-</code> in its name):
+
+```
+kubectl cp test_data.cql k8ssandra-dc1-default-sts-0:/tmp -n k8ssandra -c cassandra
+```
+
+Now extract the password to access Cassandra with the k8ssandra-superuser (the password is different for each installation unless it is explicitly set at install time): 
+
+```
+% kubectl get secret k8ssandra-superuser -n k8ssandra -o jsonpath="{.data.password}" | base64 --decode ; echo
+
+XHsZ943WBg5RPNhVAT8x
+```
+
+Let’s now run the uploaded cql script and check that you can read the data (don’t forget to replace the password with the one extracted in the previous step):
+
+```
+% kubectl exec -it k8ssandra-dc1-default-sts-0 -n k8ssandra -c cassandra -- cqlsh -u k8ssandra-superuser -p XHsZ943WBg5RPNhVAT8x -f /tmp/test_data.cql
+
+% kubectl exec -it k8ssandra-dc1-default-sts-0 -n k8ssandra -c cassandra -- cqlsh -u k8ssandra-superuser -p XHsZ943WBg5RPNhVAT8x -e "SELECT * FROM medusa_test.users"
+
+ email             | name          | state
+-------------------+---------------+-------
+ alice@example.com |   Alice Smith |    TX
+   bob@example.com |     Bob Jones |    VA
+ david@example.com |    David Yang |    NV
+ carol@example.com | Carol Jackson |    CA
+
+(4 rows)
+```
+
+Now backup this data, and check that files get created in your MinIO bucket. 
+
+To that end, use the following command:
+
+```
+helm install my-backup k8ssandra/backup -n k8ssandra --set name=backup1,cassandraDatacenter.name=dc1
+```
+
+Since the backup operation is asynchronous, you can monitor its completion by running the following command:
+
+```
+kubectl get cassandrabackup backup1 -n k8ssandra -o jsonpath={.status.finishTime}
+```
+
+As long as this doesn’t output a date and time, then the backup is still running. With the amount of data present and the fact that you’re using a locally accessible backend, this should complete quickly.
+
+Now refresh the MinIO UI and you should see some files in the k8ssandra-medusa bucket:
+
+![K8ssandra Backup Files](k8ssandra-medusa-backup.png)
+
+An index folder should appear (it is Medusa’s backup index) and then another folder that is specific to each Cassandra node in the cluster (in this case there is only one node).
+
+# Deleting the data and restoring the backup
+
+Now delete the data by truncating the table, and check that the table is empty:
```

Comment:
Should we mention anything about snapshots?

Reply:
I think that would add cognitive load in reading the blog post. We're just trying to show that restoring backups does what we want and talking about how you could restore data using the snapshot would be confusing. 
If we want to dodge that bullet, maybe the blog post should instead perform deletions rather than a truncate?

Reply:
I agree. It's tangential at best and really doesn't add much value. Thanks for the feedback.